style.css

```diff
@@ -0,0 +1,328 @@
+}
+
+.contact_link:hover:before {
+  content: "";
+  background-size: cover;
+  height: 30px;
+  width: 30px;
+  border-radius: 50%;
+  margin-right: 15px;
+  margin-left: 5px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.gh:hover:before {
+  background-image: url(images/github_icon.png);
+}
+
+.tel:hover:before {
+  background-image: url(images/phone_icon.png);
+}
+
+.ln:hover:before {
+  background-image: url(images/linked_icon2.png);
+}
+
+.mail:hover:before {
+  background-image: url(images/mail_icon.png);
+}
+
+section {
+  padding: 10px 20px 10px 20px;
+}
+
+a {
+  text-decoration: none;
+  color: royalblue;
+}
+
+footer {
+  text-align: center;
+  flex-direction: row;
+}
+
+.skills_lang {
+  background-color: #fde4cf;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: flex-start;
+  align-content: stretch;
+  flex-wrap: wrap;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.education_courses {
+  background-color: #fde4cf;
+  padding-left: 20px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-right: 10px;
+}
+
+.link_list {
+  list-style-type: none;
+}
+
+.link_contaner {
+  margin-top: 20px;
+}
+
+.wrapper {
+  padding: 20px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap-reverse;
+}
+
+footer {
+  padding-bottom: 10px;
+}
+
+.summary {
+  padding: 20px;
+}
+
+@media screen and (max-width: 996px) {
+  .skills_lang {
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-evenly;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .languages {
+    text-align: left;
+  }
+
+  .education_courses {
+    width: 100%;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .name {
+    font-size: 20px;
+  }
+
+  h3 {
+    margin: 10px;
+  }
+
+  h2.position {
+    margin: 15px;
+
+    margin-left: 20%;
+  }
+
+  .wrapper {
+    padding: 0;
+  }
+
+  .summary {
+    padding: 0;
+  }
+
+  .education_courses {
+    padding: 0 10px 0 10px;
+  }
+
+  .skills_lang {
+    padding: 0 10px 0 10px;
+  }
+
+  .section {
+    padding: 0;
+  }
+
+  .position {
+    font-size: 16px;
+    width: 60%;
+  }
+
+  .link_contaner {
+    margin: 10px;
+  }
+
+  body {
+    font-size: 15px;
+  }
+  .skills_lang {
+    width: 100%;
+    flex-direction: row;
+    justify-content: space-evenly;
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .link_list {
+    margin: 0;
+    padding: 0;
+  }
+
+  img.photo {
+    width: 90%;
+    margin: 0 auto;
+  }
+}
+
+@media screen and (max-width: 360px) {
+  .contact_link {
+    font-size: 12px;
+  }
+
+  .skills_lang {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact_link.ln:before {
+    margin: 1px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .contact_link.gh:before {
+    margin: 1px;
+    width: 15px;
+    height: 15px;
+  }
+
+  .contact_link.tel:before {
+    margin: 1px;
+    width: 15px;
+    height: 15px;
+  }
```

Comment:
Looks lie same styling for .contact_link:before
Can be shortend to 
```
 .contact_link:before {
    margin: 1px;
    width: 15px;
    height: 15px;
  }
```

By the way, 1px margin is too small, you can use 5-10 easily. Especially at the bottom, on the right side 
![image](https://user-images.githubusercontent.com/20583676/96559383-a752a280-12c5-11eb-87f8-355c4b64e129.png)

Reply:
Thank for advice